packages/devtools_app/lib/src/inspector/inspector_tree_flutter.dart

```diff
@@ -591,5 +613,52 @@ class InspectorRowContent extends StatelessWidget {
         ),
       ),
     );
+
+    // Wrap with error tooltip/marker if there is an error for this node's widget.
+    return errorText == null
+        ? rowWidget
+        : Stack(
+            children: [
+              rowWidget,
+              ErrorIndicator(
+                errorText: errorText,
+                indent: currentX - rowHeight * 2,
+              ),
+            ],
+          );
+  }
+}
+
+class ErrorIndicator extends StatelessWidget {
+  const ErrorIndicator({
+    Key key,
+    @required this.errorText,
+    @required this.indent,
+  }) : super(key: key);
+
+  final String errorText;
+  final double indent;
+
+  @override
+  Widget build(BuildContext context) {
+    return Tooltip(
+      message: errorText,
+      child: Padding(
+        padding: EdgeInsets.only(
+          left: indent,
+          top: (rowHeight - defaultIconSize) / 2,
+        ),
+        child: SizedBox(
+          width: defaultIconSize,
+          height: defaultIconSize,
+          child: Container(
```

Comment:
instead of wrapping this container in a SizedBox and Padding, just use the `width`, `height`, and `padding` properties on `Container`

Reply:
Done for width/height, though not padding as that causes the `BoxDecoration` to render in the wrong place (we need it to be inset by the padding, but if the padding is on the Container, the BoxDecoration doesn't consider it).